examples/label_studio_annotation/README.md

```diff
@@ -0,0 +1,318 @@
+# 🏷 Continuous Annotation with Label Studio
+
+Data annotation / labeling is a core part of MLOps that is frequently left out
+of the conversation. ZenML offers a way to build continuous annotation (combining
+training and annotation into a loop) with Label Studio. This uses a combination
+of user-defined steps as well as some built-in steps that ZenML provides.
+
+## Basic guide to running this example
+
+This example requires a few more steps than usual, given that it requires a
+cloud artifact store. Instructions for Azure, AWS S3 and GCP can be found in the
+following guide.
+
+## 🗺 Overview
+
+NOTE: This example currently only runs with a cloud artifact store. [This guide
+by Label Studio](https://labelstud.io/guide/storage.html) contains more details
+if you have trouble understanding the instructions below.
```

Comment:
This feels a bit redundant, would propose to remove the second paragraph and move the first one down:

```suggestion
## 🗺 Overview

This example requires a few more steps than usual, given that it requires a
cloud artifact store. Instructions for Azure, AWS S3 and GCP can be found in the
following guide.
```